reactivesocket-tck-drivers/src/main/java/io/reactivesocket/tckdrivers/client/JavaClientDriver.java

```diff
@@ -84,6 +84,13 @@ public void runTests() throws IOException {
             }
             line = reader.readLine();
         }
+        // "priming' server, seems to reduce the frequency of random errors
```

Comment:
I was actually wrong about this line of code, but one of my other changes did get rid of random errors so they are virtually nonexistent now (before I was doing things in an order where the chance of a data race was extremely high). IMO because it's a network protocol there is a > 0% chance of something failing somewhere, but you're right, I phrased this incorrectly, and will change it.